config/environments/production.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
```

Comment:
Is this just here so as not to worry about it in future migrations?

Reply:
Even though production.rb isn't used directly, it's normally present in a new app and the deployed server environment files are based on its settings, so I added it (since it was missing before) so that when we run the `rails app:update` task in the future we'll get a diff of what new production settings Rails recommends.  It should make upgrades a little easier.

(and I added `#frozen_string_literal: true` to be consistent with the other environment files)